<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudKernRulePage</class>
 <widget class="QWidget" name="AudKernRulePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_13">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>391</width>
     <height>301</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>800</height>
    </size>
   </property>
   <property name="title">
    <string>内核审计参数</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>允许的未完成审计缓冲区的最大数目：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QLineEdit" name="max_noaud_buf_lineEdit">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>内核审计</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="enable_radioButton">
      <property name="text">
       <string>启用</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QRadioButton" name="disable_radioButton">
      <property name="text">
       <string>禁用</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>设置通知内核处理关键错误的标识: </string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QRadioButton" name="silent_radioButton">
        <property name="text">
         <string>无动作</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="printk_radioButton">
        <property name="text">
         <string>通知</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="panic_radioButton">
        <property name="text">
         <string>混乱</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>当从一个文件中读取规则忽略错误:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QRadioButton" name="audctl_i_radioButton">
      <property name="text">
       <string>忽略</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QRadioButton" name="audctl_c_radioButton">
      <property name="text">
       <string> 不忽略</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>以每秒钟消息条数为单位的速率限制:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3" colspan="2">
     <widget class="QLineEdit" name="rate_limit_lineEdit">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="apl_kern_paramButton">
      <property name="text">
       <string>应用</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2" colspan="3">
     <widget class="QPushButton" name="ker_aud_param_saveButton">
      <property name="text">
       <string>保存到文件</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
